Add Alex's new publication (#122)
volkamerlab/volkamerlab_org: content/blog/virtual_control_group.md

---

> title: "New paper published: presenting hurdles and signposts on the road to virtual control groups!"

Reviewer: ```suggestion
title: "New paper presenting hurdles and signposts on the road to virtual control groups!"
```

date: 2023-04-20T14:27:59+01:00
author: alexander.gujarnov
---

> Have a look on our newest publication presenting a method to generate and validate Virtual Control Groups (VCGs) in systemic toxicity studies [here](https://www.frontiersin.org/articles/10.3389/fphar.2023.1142534/full). This paper highlights the impact of anesthetics on measured electrolytes in animals and presents how these anesthetic protocols can affect the performance of virtual control groups.

Reviewer: ```suggestion
Have a look on our [newest publication](https://www.frontiersin.org/articles/10.3389/fphar.2023.1142534/full) presenting a method to generate and validate Virtual Control Groups (VCGs) in systemic toxicity studies. This paper highlights the impact of anesthetics on measured electrolytes in animals and presents how these anesthetic protocols can affect the performance of virtual control groups.
```

> This article was published by out PhD student at Bayer Pharmaceuticals as a member of the the eTRANSAFE Virtual Control Groups team. 

Reviewer: ```suggestion
This article was published by Alexander, our PhD student at Bayer Pharmaceuticals as a member of the eTRANSAFE Virtual Control Groups team. 
```
